examples/UnionExample.hs

```diff
@@ -0,0 +1,32 @@
+module UnionExample where
```

Comment:
Hm, what's the best way to do that? It's not test code, and the only thing we care about is that it compiles.